include/stl2/view/move.hpp

```diff
@@ -0,0 +1,83 @@
+// cmcstl2 - A concept-enabled C++ standard library
+//
+//  Copyright Casey Carter 2016
+//
+//  Use, modification and distribution is subject to the
+//  Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at
+//  http://www.boost.org/LICENSE_1_0.txt)
+//
+// Project home: https://github.com/caseycarter/cmcstl2
+//
+#ifndef STL2_DETAIL_VIEW_MOVE_HPP
+#define STL2_DETAIL_VIEW_MOVE_HPP
+
+#include <stl2/detail/fwd.hpp>
+#include <stl2/detail/concepts/object.hpp>
+#include <stl2/detail/iterator/move_iterator.hpp>
+#include <stl2/detail/range/concepts.hpp>
+
+STL2_OPEN_NAMESPACE {
+	template <InputRange Rng>
+	class move_view {
```

Comment:
Why name this class _view? I would think of it more like an adaptor, that change the behavior of the wrapped range.